modules/local/bbmaprepair/main.nf

```diff
@@ -0,0 +1,49 @@
+process BBMAPREPAIR {
+    tag "$meta.id"
+    label 'process_single'
+
+    container 'docker.io/mazzalab/fastqrepair_nf_env:1.0.1'
```

Comment:
Is there a possibility to use a seqera container and/or biocontainers?

Reply:
To the best of my knowledge, there is neither a seqera container nor a biocontainer for this tool.
Since this is a critical point to make the pipeline work with conda as well, I think that I have to make that. Isn't it?
What do you suggest?